.github/workflows/stage-release.yml

```
@@ -47,6 +47,12 @@ jobs:
           git config user.name "${{ github.actor }}"
           git config user.email "${{ github.actor_id }}+${{ github.actor }}@users.noreply.github.com"
 
+      - name: Do custom changes
+        run: |
+          if [ -f "./bin/do-custom-changes.sh" ]; then
+            ./bin/do-custom-changes.sh ${{ steps.bump_version.outputs.version }}
+          fi
```

I think it would be better if we completely skip running this step if the file doesn't exist. I found the [expression `hashFiles(path)`](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#hashfiles) which we can use for that purpose:
```suggestion
      - name: Do custom changes
        if: hashFiles('./bin/do-custom-changes.sh') != ''
        run: ./bin/do-custom-changes.sh ${{ steps.bump_version.outputs.version }}
```